### ISSUE TITLE
README is confusing

### ISSUE THREAD

**Issue by nobodyatandnothing** [Biont/sway-launcher-desktop#62]
`--class=launcher`: --class does not work for all terminals. for instance in foot it does not work and instead -a/--app-id may be used

**Comment by Biont** [comment]
Good point. IIRC this section was not always suggesting `$term` but actually referred to my then-preferred `urxvt`. Now this might not make so much sense any more so I simply removed it for brevity
